contrib/natss/pkg/dispatcher/main.go

```diff
@@ -68,6 +68,8 @@ func main() {
 	if err != nil {
 		logger.Fatal("Unable to create NATSS dispatcher.", zap.Error(err))
 	}
+	// Starting Connect to establish connection with NATS
+	go dispatcher.Connect()
```

Comment:
My preference is to put this inside the dispatcher's `Start(stopCh)`.

Reply:
ok, I will give it a try.

Reply:
I don't see this change in, did it not work?

Reply:
done